<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reflMainWidget</class>
 <widget class="QMainWindow" name="reflMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISIS Reflectometry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
  <layout class="QVBoxLayout" name="layoutMain">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="labelPrototype">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>PROTOTYPE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitterTables">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="groupSearchPane">
      <property name="title">
       <string>Search Runs</string>
      </property>
      <layout class="QVBoxLayout" name="layoutSearchPane">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <layout class="QFormLayout" name="layoutSearchForm">
         <item row="0" column="0">
          <widget class="QLabel" name="labelInstrument">
           <property name="text">
            <string>Instrument:</string>
           </property>
           <property name="buddy">
            <cstring>comboSearchInstrument</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboSearchInstrument">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select the instrument to search with</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>RB Search:</string>
           </property>
           <property name="buddy">
            <cstring>textRB</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="textRB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Investigation to search for</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="buttonSearch">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Search</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableRuns">
         <property name="toolTip">
          <string>Search results</string>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Run #</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutSearchBottomRow">
         <item>
          <spacer name="spacerTransfer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTransfer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Add the selected run to the table</string>
           </property>
           <property name="text">
            <string>Add Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupProcessPane">
      <property name="title">
       <string>Process Runs</string>
      </property>
      <layout class="QHBoxLayout" name="layoutTableView">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layoutButtonsTable">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="layoutTableButtons">
           <item>
            <widget class="QPushButton" name="buttonNew">
             <property name="toolTip">
              <string>Start a new table</string>
             </property>
             <property name="text">
              <string>New Table</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonSave">
             <property name="toolTip">
              <string>Save the current table as a Table Workspace</string>
             </property>
             <property name="text">
              <string>Save Table</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonSaveAs">
             <property name="toolTip">
              <string>Save the current table as a Table Workspace</string>
             </property>
             <property name="text">
              <string>Save Table As</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerTableButtons">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelWorkspaceSelector">
             <property name="text">
              <string>Table:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="workspaceSelector">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Select the Table Workspace to display</string>
             </property>
             <property name="WorkspaceTypes" stdset="0">
              <stringlist>
               <string>TableWorkspace</string>
              </stringlist>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="viewTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Runs to process</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ContiguousSelection</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutRowButtons">
           <property name="margin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="buttonAddRow">
             <property name="toolTip">
              <string>Add a row beneath the selected row(s)</string>
             </property>
             <property name="text">
              <string>Add Row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonDeleteRow">
             <property name="toolTip">
              <string>Delete selected row(s) from the table</string>
             </property>
             <property name="text">
              <string>Delete Row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonGroupRows">
             <property name="text">
              <string>Group Rows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar"/>
           </item>
           <item>
            <widget class="QLabel" name="labelProcessInstrument">
             <property name="text">
              <string>Instrument:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboProcessInstrument">
             <property name="toolTip">
              <string>Select the instrument to assume for fetching runs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonProcess">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Process the selected rows</string>
             </property>
             <property name="text">
              <string>Process</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboSearchInstrument</tabstop>
  <tabstop>tableRuns</tabstop>
  <tabstop>viewTable</tabstop>
  <tabstop>buttonProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
